include/graaflib/algorithm/coloring/welsh_powell.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <graaflib/graph.h>
+#include <graaflib/types.h>
+
+namespace graaf::algorithm {
+
+/**
+  * @brief Applies the Welsh-Powell greedy graph coloring algorithm to the given graph.
+  * 
```

Comment:
```suggestion
 * @brief Applies the Welsh-Powell greedy graph coloring algorithm to the given graph.
 * 
```

Reply:
I presume in this you want me to make the alignment of the lines same as the other code files(greedy_graph_coloring.h)!!!